training_pipeline/models/model.py

```diff
@@ -187,7 +152,7 @@ def _build_model(num_labels: int) -> tf.keras.Model:
     model.compile(
         optimizer=Adam(learning_rate=_LR),
         loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True),
-        metrics=["accuracy"],
+        metrics=["sparse_categorical_accuracy"],
```

Comment:
If you supply `accuracy` Keras will automatically infer the appropriate variant.